wagtail/admin/templates/wagtailadmin/pages/edit.html

```diff
@@ -15,9 +15,22 @@
     </div>
 
     {% block form %}
+
         <form id="page-edit-form" action="{% url 'wagtailadmin_pages:edit' page.id %}" method="POST" novalidate{% if form.is_multipart %} enctype="multipart/form-data"{% endif %} data-edit-form>
             {% csrf_token %}
 
+            {# Note that the position of the button within the form allows the js to collapse / expand only those panels within the same form #}
+            <button
+                type="button"
+                class="button button-small button-secondary"
+                data-all-panels-toggle
+                data-expand-text="{% trans 'Expand all sections' %}"
+                data-collapse-text="{% trans 'Collapse all sections' %}"
+                aria-expanded="true"
+            >
+                    {% trans "Collapse all sections" %}
+            </button>
```

Comment:
Since this is repeated twice identically, this feels worth having its own template partial.